docs/rules/prefer-object-spread.md

```diff
@@ -14,6 +14,7 @@ Object.assign({}, foo)
 
 Object.assign({}, {foo: 'bar'})
 
+// Set a default value for "foo".
```

Comment:
I don't think this accurately describes the code. The first argument is really the receiver and return value, so it's the object you want to operate on. `foo` will either be overwritten or remain the same, but I don't think "default" is clear in this context.

Reply:
i was describing the idiomatic usage rather than precisely describe the semantics.  what phrasing would you suggest instead ?